### ISSUE TITLE
Feature: Allow pausing operations in Status Center

### ISSUE THREAD

**Issue by Josh65-2201** [files-community/Files#14929]
### What feature or improvement do you think would benefit Files?

*

### Requirements

- Add a paused/resume button under the cancel button.
   - This could lead to accidently cancelling operations so a prompt when clicked could be added. Alternatively move the pause/resume button as far down as possible in the UI.
- Set graph to yellow when paused to help show status.

### Files Version

3.2.6.0

### Windows Version

10.0.22621.3155

**Comment by yaira2** [comment]
We'll need a design but otherwise this is ready to build.

**Comment by yair100** [comment]
As suggested in #17571, we can add the pause and resume option to the overflow menu.

<img width="626" height="222" alt="Image" src="https://github.com/user-attachments/assets/c6fda55e-cd08-4414-9d19-a4b8c62f7812" />

**Comment by YA-FACE-FLAT** [comment]
@yaira2 I agree with the covered topic in #17571 but make sure that the bar turns yellow, as well as the icon on pause, and the design for the cancelation prompt should be based on the design for similar win11 styled-dialogs, other improvements include make the cancel button have red text and symbol. Example for reference: 

<img width="525" height="117" alt="Image" src="https://github.com/user-attachments/assets/60d83795-1eb3-45e9-ad1b-da99800a899d" />
